v2/cmd/controller/main.go

```diff
@@ -17,7 +17,7 @@ import (
 
 func main() {
 	setupLog := ctrl.Log.WithName("setup")
-	ctrl.SetLogger(klogr.New())
+	ctrl.SetLogger(klogr.New()) //nolint: staticcheck
```

Comment:
We have a separate item tracking fixing this, and doing so in this PR was nontrivial so I skipped it.